src/backend/src/transformers/work-packages.transformer.ts

```diff
@@ -33,6 +34,7 @@ const workPackageTransformer = (wpInput: Prisma.Work_PackageGetPayload<WorkPacka
       detail: change.detail,
       dateImplemented: change.dateImplemented
     })),
+    teamTypes: wpInput.project.teams.map(teamTransformer).map((team) => team.teamType!),
```

Comment:
we never ever want to force unwrap. This will lead to unexpected results in typescript, and will cause crashes in other programming languages. You always want to either conditionally unwrap or use a null coelescing operator i.e ??.